app/services/atmosphere/cloud/scale_appliance.rb

```diff
@@ -0,0 +1,51 @@
+module Atmosphere
+  module Cloud
+    class ScaleAppliance
+      def initialize(appliance, quantity)
+        @appliance = appliance
+        @quantity = quantity
+      end
+
+      def execute
+        if optimization_strategy.can_scale_manually?
+          action.log(I18n.t('scale_appliance.start'))
+          if quantity > 0
+            vms = optimization_strategy.vms_to_start(quantity)
+            appl_manager.start_vms!(vms)
+          else
+            if appliance.virtual_machines.count > quantity.abs
+              vms = optimization_strategy.vms_to_stop(-quantity)
+              appl_manager.stop_vms!(vms)
+            else
+              appl_manager.unsatisfied(I18n.t('scale_appliance.to_small_vms'))
+            end
+          end
+          action.log(I18n.t('scale_appliance.end'))
+        else
+          action.log('Scaling not allowed')
```

Comment:
Please be more precise - tell the user that `manual` scaling is not allowed (and try to tell why).